docs/getting-started/my-account/README.md

```diff
@@ -81,7 +81,7 @@ If you want to help us to add further languages or to add the translations in yo
 
 You can choose a time zone in which you work and how the comments and changes will be saved.
 
-![time zone](1572882881007.png)
+![my-account-change-time-zone](Bildschirmfoto 2022-03-02 um 14.47.20.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.